Dockerfile

```diff
@@ -44,6 +45,20 @@ FROM --platform=$BUILDPLATFORM $ERASERBASEIMAGE as eraser
 COPY --from=eraser-build /workspace/out/eraser /
 ENTRYPOINT ["/eraser"]
 
+FROM builder AS collector-build
+
+ARG TARGETOS
```

Comment:
do we need to repeat these args? i think i added this to @pmengelbert's pr review